primitives/src/traits.rs

```diff
@@ -69,3 +76,20 @@ pub trait SimpleAssetSystem {
 	/// Get the default asset/currency ID in the system
 	fn default_asset_id() -> Self::AssetId;
 }
+
+/// Something which can perform reward payment to staked validators
+pub trait StakerRewardPayment {
```

Comment:
There is a potential for "primitives" to become a place for holding random unrelated stuff.

Reply:
that's what it's used for, we could rename it to something besides `primitives` to indicate that better?